imhotep/reporters/printing.py

```diff
@@ -1,5 +1,4 @@
 import logging
-from six import string_types
```

Comment:
PyCharm called it out as unused, and it appears to be correct. (Unless it has some side effect I'm ignorant of.) Happy to put it back!